ui/partials/perf/comparechooserctrl.html

```diff
@@ -21,11 +27,19 @@
         <div class="panel-heading">New</div>
         <div class="panel-body">
           <label for="new-project-selector">Project</label>
-          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
+          <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects" ng-change="updateNewRevisionTips()"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group" ng-class="{'has-error': newRevisionError}">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+            <input id="new-revision-input" maxlength="12" class="form-control" type="text" ng-model="newRevision"/>
+            <div class="input-group-btn">
+              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              Pushes<span class="caret"></span></button>
```

Comment:
I think I prefer the term `Recent` to `Pushes`, as it's more precise.

Also, there should be a space between the text and the caret. I.e.

    Pushes <span class="caret"/>